pkg/db/columnhandler_test.go

```diff
@@ -161,7 +161,7 @@ func splitAndTrimCsv(csv string) []string {
 	return result
 }
 
-//mock DB entity used to test the validation logic
+// mock DB entity used to test the validation logic
```

Comment:
https://go.dev/doc/comment
> [...] doc comments for types start with complete sentences naming the declared symbol. An explicit subject often makes the wording clearer, and it makes the text easier to search, whether on a web page or a command line

Reply:
Fully agree on your points. But a fix would need to change actually all files which use comments to be consistent again. 

As we are in maintenance mode and not changing the old reconciler any more (replacement happens in coming months), we should avoid bigger code changes as the efforts will get lost soon.

Please excuse when I recommend to leave the wrong comment style and focus on clean comments in the new reconciler code bases.